inc/cleantalk-settings.php

```diff
@@ -2061,6 +2061,27 @@ function apbct_settings__validate($settings)
         return $settings;
     }
 
+    // Ajax type
+    $available_ajax_type = apbct_settings__get_ajax_type();
+
+    if (!isset($apbct->data['ajax_type'])) {
```

Comment:
1. No way that **$apbct->data['ajax_type']** could be not set.
2. Are you using $apbct->data to store a setting? Can we use just a single setting? Otherwise, it is some kind of subsetting here.